linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_remote_access_monitoring/ansible/ubuntu.yml

```diff
@@ -0,0 +1,43 @@
+# platform = multi_platform_ubuntu
```

Comment:
I believe you can adopt the `shared.yml` instead of creating a new file specific for one product.

Reply:
we already have a specific bash remediation because of differences between distros.
That doesn't mean the we couldn't put all in one file and do some jinja logic but it does make maintenance a bit easier having separate files

Reply:
Understood. Thanks for the update. If it would make things more difficult, it's not necessary. All good.
But it's good to keep this in mind to reduce duplication, make rules simpler and reusable whenever possible.